superset/config.py

```diff
@@ -1397,6 +1397,7 @@ def EMAIL_HEADER_MUTATOR(  # pylint: disable=invalid-name,unused-argument
     },
     "content_security_policy_nonce_in": ["script-src"],
     "force_https": False,
+    "session_cookie_samesite": "None",
```

Comment:
just testing if this works.. the long term solution would be to put this in the github workflow above.